plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php

```diff
@@ -220,6 +220,9 @@ protected function add_general_group_blocks() {
 					'property' => 'description',
 					'label'    => __( 'Note', 'woocommerce' ),
 					'help'     => 'Enter an optional note displayed on the product page when customers select this variation.',
+					'lock'     => array(
+						'move' => true,
+					),
```

Comment:
This prevents the moving options from showing in the description toolbar ( similar changes below for the summary block also ).